test/integration/patients/patient/dashboard.js

```diff
@@ -552,6 +568,18 @@ context('patient dashboard page', function() {
           program: getRelationship(testProgramIds[1], 'programs'),
         },
       }),
+      getFlow({
+        attributes: {
+          name: 'Should not show - not visible to current user team',
+          behavior: 'standard',
+          published_at: testTs(),
+          archived_at: null,
+        },
+        relationships: {
+          program: getRelationship(testProgramIds[1], 'programs'),
+          visible_to: { data: [{ type: 'teams', id: teamNurse.id }] },
```

Comment:
Better way to do this?

Reply:
```suggestion
          visible_to: { data: [{ type: teamNurse.type, id: teamNurse.id }] },
```

Does that property exist?

Reply:
Yep that works 👍 . Via `getRelationship([teamNurse])`. I forgot we had examples of doing that elsewhere in our tests.